packages/grpc-js/prettier.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  proseWrap: 'always',
+  singleQuote: true,
+  trailingComma: 'es5',
```

Comment:
@murgatroid99 This is a default value, however we might want to go with `all` here to reduce noise in PRs. What's your take on this?

Reply:
I don't really like how code looks with the `all` value, and it's not entirely clear that it would result in valid TypeScript.

Reply:
It most definitely results in valid TypeScript, as it's also a valid ECMAScript. ES6 added support for trailing commas (e.g. last property in the object). I generally tend to use `all`, as it gives cleaner PRs (if you add a property, only one line is changed, not two - current and previous with the comma).

I don't have a strong opinion on this tho, just a personal preference, so we can leave it as `es5` if you will.

Reply:
The documentation I found says that the `es5` option already adds commas after the last item in objects and arrays, and that the `all` option puts it in other places, including after the last function argument, which looks weird to me.